src/main/java/org/broadinstitute/barclay/help/BashTabCompletionDoclet.java

```diff
@@ -0,0 +1,533 @@
+                callerScriptPrefixMutexArgs = quoteEachWord(option[1]);
+                hasParsedOption = true;
+            }
+            else if (option[0].equals(CALLER_SCRIPT_PREFIX_ALIAS_ARGS)) {
+                // We have to format this option to contain quotes around each group of options:
+                callerScriptPrefixAliasArgs = quoteEachWord(option[1]);
+                hasParsedOption = true;
+            }
+            else if (option[0].equals(CALLER_SCRIPT_PREFIX_ARG_MIN_OCCURRENCES)) {
+                callerScriptPrefixMinOccurrences = option[1];
+                hasParsedOption = true;
+            }
+            else if (option[0].equals(CALLER_SCRIPT_PREFIX_ARG_MAX_OCCURRENCES)) {
+                callerScriptPrefixMaxOccurrences = option[1];
+                hasParsedOption = true;
+            }
+
+            else if (option[0].equals(CALLER_SCRIPT_POSTFIX_LEGAL_ARGS)) {
+                callerScriptPostfixLegalArgs = option[1];
+                hasCallerScriptPostfixArgs = !callerScriptPostfixLegalArgs.isEmpty();
+                hasParsedOption = true;
+            }
+            else if (option[0].equals(CALLER_SCRIPT_POSTFIX_ARG_VALUE_TYPES)) {
+                // We have to format this option to contain quotes around each word:
+                callerScriptPostfixArgValueTypes = quoteEachWord(option[1]);
+                hasParsedOption = true;
+            }
+            else if (option[0].equals(CALLER_SCRIPT_POSTFIX_MUTEX_ARGS)) {
+                // We have to format this option to contain quotes around each word:
+                callerScriptPostfixMutexArgs = quoteEachWord(option[1]);
+                hasParsedOption = true;
+            }
+            else if (option[0].equals(CALLER_SCRIPT_POSTFIX_ALIAS_ARGS)) {
+                // We have to format this option to contain quotes around each word:
+                callerScriptPostfixAliasArgs = quoteEachWord(option[1]);
+                hasParsedOption = true;
+            }
+            else if (option[0].equals(CALLER_SCRIPT_POSTFIX_ARG_MIN_OCCURRENCES)) {
+                callerScriptPostfixMinOccurrences = option[1];
+                hasParsedOption = true;
+            }
+            else if (option[0].equals(CALLER_SCRIPT_POSTFIX_ARG_MAX_OCCURRENCES)) {
+                callerScriptPostfixMaxOccurrences = option[1];
+                hasParsedOption = true;
+            }
+        }
+
+        return hasParsedOption;
+    }
+
+    @Override
+    protected void processWorkUnitTemplate(
+            final Configuration cfg,
+            final DocWorkUnit workUnit,
+            final List<Map<String, String>> indexByGroupMaps,
+            final List<Map<String, String>> featureMaps) {
+
+            // For the Bash Test Doclet, this is a no-op.
+            // We only care about the index file.
+    }
+
+    /**
+     * The Index file in the Bash Test Doclet is what generates the actual tab-completion script.
```

Comment:
Removed the .template part of the name.

Replaced "Test" in comment with "Completion"